tuned/plugins/plugin_kthread.py

```
@@ -0,0 +1,736 @@
+		self._instance_count = 0
+
+		try:
+			self._scheduler_utils = SchedulerUtils()
+		except AttributeError:
+			self._scheduler_utils = SchedulerUtilsSchedutils()
+
+		global have_perf
+		if have_perf:
+			try:
+				self._perf_setup()
+			except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)